common/app/views/fragments/atoms/youtube.scala.html

```diff
@@ -77,7 +77,7 @@
                 </div>
             }
             @bestPosterImage.map { image =>
-                <div class="@RenderClasses(Map("youtube-media-atom__overlay" -> true, "vjs-big-play-button" -> !expired))" style="background-image: url(@Video700.bestSrcFor(image))">
+                <div class="@RenderClasses(Map("youtube-media-atom__overlay" -> true, "vjs-big-play-button" -> !expired))" style="background-image: url(@Video700.bestSrcFor(image))" role="button">
```

Comment:
I think this is adding noise and not achieving any real value, I’ll probably back it out